code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -136,8 +135,9 @@
 	active_force = 40
 	active_throwforce = 60
 	active_w_class = ITEMSIZE_HUGE
-	force = 20
-	throwforce = 30
+	force = 30
+	throwforce = 40
+	armor_penetration = 20 
```

Comment:
either this value or the PR description and changelog should be changed, as they don't match up.